### PR TITLE
chore: update elasticsearch to 8.8.2 and lucene to 9.6.0

### PR DESCRIPTION
**Pull request by israelsimas**
Bump latest version of ES 8.7.1 to 8.8.2 and lucene of 9.5.0 to 9.6.0

### CODE REVIEW

**Comment by israelsimas** [comment]
@nathancday this bump to ES 8.8.2

**Comment by rtancman** [comment]
@israelsimas nice! @worleydl Look this :)

File: src/javaRestTest/java/com/o19s/es/ltr/logging/LoggingIT.java [hunk 1]
```diff
@@ -623,7 +623,7 @@ protected void assertSearchHitsExtraLogging(Map<String, Doc> docs, SearchRespons
             Doc d = docs.get(hit.getId());
 
             assertEquals(6, log1.size());
-            assertEquals(6, log2.size());
```
Comment (wrigleyDan) [inline src/javaRestTest/java/com/o19s/es/ltr/logging/LoggingIT.java:626]: This test fails for me with the change in this PR.
Any specific reason that this change was necessary for you?

**Review by wrigleyDan — COMMENTED** [review]
One test error resulting from the change in one of the test classes. Other than that: looks good to me!

If we can resolve that last thing, we're good to create a release I guess.

**Comment by wrigleyDan** [comment]
Merged as part of another PR, release built and published, thanks for the contribution!

**Pull request closed without merging**